templates/loader.ejs

```diff
@@ -51,9 +51,18 @@
     var keyset = app && app.privateData.keyset;
     keyset && keyset.setValue(keyset.NAVIGATION);
     document.onkeydown = function(event) {
-      if (event.keyCode === KeyEvent.VK_ENTER) message('vkenter', cb);
-      if (event.keyCode === KeyEvent.VK_LEFT) message('vkleft');
-      if (event.keyCode === KeyEvent.VK_RIGHT) message('vkright');
+      if (event.keyCode === KeyEvent.VK_ENTER) {
+        message('vkenter', cb);
+        event.preventDefault();
+      }
+      if (event.keyCode === KeyEvent.VK_LEFT) {
+        message('vkleft');
+        event.preventDefault();
+      }
+      if (event.keyCode === KeyEvent.VK_RIGHT) {
+        message('vkright');
+        event.preventDefault();
```

Comment:
So this was the fix, I suppose @aldafu ?

Reply:
For those ok. devices yes. To be clean this should always have been used, but only the ok. device had problems if it didn't exist.